### DISCUSSION TITLE
Cannot copy paste between mac m1 <---> Linux 22.04.3

### DISCUSSION THREAD

**Discussion by r1axodn** [utmapp/UTM#6153] [Linux]
Issue: I'm currently running 22.04.3 ubuntu on a mac m1, but I can't copy and paste between the host(mac) and VM(linux) doesn't work. How can I fix it? I've tried installing spice-vdagent, qemu-guest-agent, and spice-webdavd, but it doesn't work.

**Configuration**  
* UTM Version: 4.3.5
* macOS Version: Ventura 13.0
* Mac Chip (Intel, M1, ...): M1

**Comment by kmatheussen** [comment]
Have you tried running the "spice-vdagent" program as a normal user?
